charts_common/lib/src/chart/common/behavior/selection/select_nearest.dart

```diff
@@ -108,7 +108,7 @@ class SelectNearest<D> implements ChartBehavior<D> {
     this.selectClosestSeries = true,
     this.eventTrigger = SelectionTrigger.hover,
     this.maximumDomainDistancePx,
-    this.hoverEventDelay,
+    this.hoverEventDelay = 100,
```

Comment:
what's this doing for us?

Reply:
ooo, i think this was from an earlier test i was doing that was trying to see if i could rate limit with that property and using hover instead of long press. it didn't work, and i forgot to reset my working set before branching

Reply:
will remove